spec/queries/atmosphere/appliances_affected_by_pmt_spec.rb

```diff
@@ -30,10 +32,10 @@
 
   context 'in development mode' do
     it 'finds affected appliances using dev_mode_property_set relation' do
-      dev_as = create(:dev_appliance_set)
-      appl1 = create(:appliance, appliance_type: at, appliance_set: dev_as)
-      create(:appliance, appliance_type: at)
+      appl1 = dev_appliance
       dev_pmt = appl1.dev_mode_property_set.port_mapping_templates.first
+      # other appliances which should not be taken into account
+      create(:appliance, appliance_type: at)
 
       affected_appl = Atmosphere::AppliancesAffectedByPmt.new(dev_pmt).find
```

Comment:
This is an implementation of Query Object taken [from this article](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/) which was created several months ago... Please in the scope of this PR focus on introduced changes. If you believe that name or location of the method is incorrect please do the change and submit another PR :smiley_cat: